packages/httpsnippet-client-api/src/index.js

```diff
@@ -282,7 +261,7 @@ module.exports = function (source, options) {
   }
 
   code
-    .push(`sdk${accessor}(${args.join(', ')})`)
+    .push(`sdk.${accessor}(${args.join(', ')})`)
```

Comment:
Moved the period out of `accessor` into here because it's cleaner that way and also without the array stuff that was on line 260 we can.